src/App.tsx

```diff
@@ -1,18 +1,42 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [currentPage, setCurrentPage] = useState(1);
+  const [limitPerPage, setLimitPerPage] = useState(5);
+
+  const amountOfItems = items.length;
+
+  const firstItem = limitPerPage * (currentPage - 1);
```

Comment:
5 * 1 - 5 = 0
5 * 2 - 5 = 5
5 * 3 - 5 = 10

```suggestion
  const firstItem = limitPerPage * currentPage - limitPerPage;
```